bruno/collections/Rafiki/Rafiki Admin APIs/Get Accounting Ledger Transfers.bru

```diff
@@ -0,0 +1,49 @@
+meta {
+  name: Get Accounting Ledger Transfers
+  type: graphql
+  seq: 1
+}
+
+post {
+  url: {{RafikiGraphqlHost}}/graphql
+  body: graphql
+  auth: none
+}
+
+body:graphql {
+  query AccountingTransfers($id: String!, $limit: Int!) {
+    accountingTransfers(id: $id, limit: $limit) {
+      debits {
+        id
+        debitAccount
+        creditAccount
+        amount
+        transferType
+        ledger
+        createdAt
+      }
+      credits {
+        id
+        debitAccount
+        creditAccount
+        amount
+        transferType
+        ledger
+        createdAt
+      }
+    }
+  }
+  
+}
+
+body:graphql:vars {
+  {
+      "id": "{{outgoingPaymentId}}"
```

Comment:
Im not able to get this to work because it doesnt find the `outgoingPaymentId`. Maybe you can tell me what I'm doing wrong?

I did the Open Payments example flow (which saves the `outgoingPaymentId` env var) and then did this and saw empty results. For good measure I tried sending this request to `{{PeerGraphqlHost}}/graphql` (happy life bank) instead but saw the same result. I went into postgres db and grabbed a `debitAccountId` from one of the `ledgerTransfers` and it worked. Should this be something other than `outgoingPaymentId` or...?

Reply:
@BlairCurrey Updated it now to be proper and always working. 🙇 

<img width="1271" alt="image" src="https://github.com/user-attachments/assets/17dac25b-afde-45d4-a500-3bbd6eac8e8b">